index.js

```diff
@@ -117,9 +134,12 @@ function createHarness(conf_) {
 
 	test.close = function () { results.close(); };
 
+	test.run = function () {};
```

Comment:
I'd advise putting any real JS functional changes into a separate PR. Even if it's a super obvious bug fix, trying to bisect or revert in the future gets to be a real pain if a big types-y PR like this one also has functional things.

This is something I've learned the hard way 🥲 https://medium.com/codecademy-engineering/converting-codecademy-to-typescript-3-learnings-and-next-steps-c5bf7465294#43c8